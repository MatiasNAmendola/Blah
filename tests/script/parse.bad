if True then 1 else if 2 fi
