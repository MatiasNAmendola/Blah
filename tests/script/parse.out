<parse> [Dk FI,Db (Oa (An (Mc (Cj (Ja (At (Tf (Fp (Ni 2))))))))),Dk IF,Dk ELSE,Dl (Ls (Se (Oa (An (Mc (Cj (Ja (At (Tf (Fp (Ni 1))))))))))),Dk THEN,Db (Oa (An (Mc (Cj (Ja (At (Tf (Fb True)))))))),Dk IF] []
